UX-Guide-Metadata/draft/principles/index.html

```diff
@@ -801,24 +801,17 @@ <h3>Legal considerations</h3>
 <p>In some jurisdictions publishers may be able to claim an exemption from the provision of accessible publications, including the provision of accessibility metadata. This should always be subject to clarification by legal counsel for each jurisdiction.
 	One example of this, at time of writing, is from the European Accessibility Act (EAA) when the publisher is a micro-enterprise (i.e., enterprises employing fewer than 10 people and with annual turnover or balance sheet total not exceeding 2 million euro).</p>
 <p>Other legal considerations currently included in the EAA may include exceptions on individual versions of a title, where jurisdiction will not require the title to be accessible if it involves a fundamental alteration of the content, or if making it accessible would place a disproportionate burden on the publisher. (This may vary in other jurisdictions).</p>
-
 <p>Publishers may need to include information about an exemption or exception in metadata for legal or clarity reasons, either to bodies that enforce legislation or for other business to business communication. However this is not information that needs to be displayed on public sites as it does not mean anything to most consumers and could lead to misunderstandings. Nevertheless, the objective is to provide as much accessibility information as is possible."</p>
-
-            <section id="legal-examples">
-					<h2>Examples</h2>
-<p>TBD</p>
-<!--
-					<aside class="example" title="Descriptive explanation">
-
-<p>This title may not yet be fully accessible to all readers. Check the list of accessibility features to determine  if this title is suitable for your way of reading.</p>
-</aside>
-<aside class="example" title="Compact explanation">
-<p>May not  be fully accessible. Check features for suitability.</p>
-</aside>
--->
+	<section id="legal-examples">
+		<h2>Examples</h2>
+		<aside class="example" title="Descriptive explanation">
+			<p>This title benefit from an exemption or exception to legal provision of accessible publication.</p>
```

Comment:
This sounds awkward, not sure what you are trying to say here. "There are no legal exceptions provided." wouldn't we just not display anything?

Reply:
Do you have a better wording? 
The intent is to say that the title benefits from an exemption.

Reply:
Maybe "This title is exempt from the legal provisions for accessible publication."

Reply:
I suggest to discuss this in the issue :) https://github.com/w3c/publ-a11y/issues/350